website/docs/index.md

```diff
@@ -14,9 +14,10 @@ hide_title: true
 [![All Contributors](https://img.shields.io/badge/all_contributors-77-orange.svg?style=flat-square)](#contributors-)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/6afbdad19e7245adbf9e9771777be3d7)](https://app.codacy.com/gh/fzyzcjy/flutter_rust_bridge?utm_source=github.com&utm_medium=referral&utm_content=fzyzcjy/flutter_rust_bridge&utm_campaign=Badge_Grade_Settings)
 
-[<img src="https://github.com/fzyzcjy/flutter_rust_bridge/raw/master/website/misc/flutter_favorite.png" width="200" />](https://flutter.dev/docs/development/packages-and-plugins/favorites)
+[<img id="flutter_favorite" src="flutter_favorite.png" width="200" />](https://flutter.dev/docs/development/packages-and-plugins/favorites)
```

Comment:
Hmm, index.md is auto generated from README.md...

Reply:
Hum... Sorry I didn't see that, so I added a `NOT EDIT` message